Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11-slim
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY app/ /app/
+
+
+EXPOSE 8000
+
+CMD ["python", "main.py"]
```

Comment:
Ensure that the `main.py` file exists in the `/app` directory as expected by the `CMD` command. This is crucial for the application to run correctly.